modules/nf-core/bcftools/concat/main.nf

```diff
@@ -0,0 +1,35 @@
+process BCFTOOLS_CONCAT {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::bcftools=1.15.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/bcftools:1.15.1--h0ea216a_0':
+        'quay.io/biocontainers/bcftools:1.15.1--h0ea216a_0' }"
+
+    input:
+    tuple val(meta), path(vcfs), path(tbi)
+
+    output:
+    tuple val(meta), path("*.gz"), emit: vcf
+    path  "versions.yml"         , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args   ?: ''
+    prefix   = task.ext.prefix ?: "${meta.id}"
+    """
+    bcftools concat \\
```

Comment:
Has there been any thoughts if it needs to sort the concatenated file afterwards? (apologies if I missed that) -- it just seems like the sensible thing to do since few vcf readers will accept unsorted vcfs, and just concat I think leaves it unsorted.

Reply:
Yes, `bcftools concat` just leaves the output-vcf-file unsorted. I added the sorting, since @amasplund wrote (https://github.com/nf-core/sarek/issues/738#issuecomment-1313829532) that that is what they have been doing.

Reply:
that sounds like a good idea